.gitignore

```diff
@@ -482,3 +482,11 @@ fabric.properties
 
 # Rhubarb Ignore
 **/rhubarb
+
+# Ignore Build/Dist Dir
+speaking-portal-project/build
```

Comment:
Odd that some files in the build folder are still being included, I wonder if the path could be off?

Reply:
I think it works now